crypto/asn1/a_gentm.c

```diff
@@ -108,7 +108,7 @@ ASN1_GENERALIZEDTIME *ASN1_GENERALIZEDTIME_adj(ASN1_GENERALIZEDTIME *s,
     return NULL;
 }
 
-const char *_asn1_mon[12] = {
+static const char *_asn1_mon[12] = {
```

Comment:
Suggestion was also to make it const, i.e. `static cost char * const`. Not just an non-const array of pointers to constant strings, but constant array of pointers to constant strings.